llpc/context/llpcCompiler.cpp

```diff
@@ -208,6 +208,65 @@ static void fatalErrorHandler(void *userData, const std::string &reason, bool ge
 #endif
 }
 
+// =====================================================================================================================
+// Returns the cache accessor object resulting from checking the caches for the glue shader for the given identifier.
+//
+// @param glueShaderIdentifier : The linker object for which the glue shaders are needed.
+// @param context : The context that contains the application caches.
+// @param compiler : The compiler object that contains the internal caches.
+static CacheAccessor checkCacheForGlueShader(StringRef glueShaderIdentifier, Context *context, Compiler *compiler) {
+  Hash glueShaderCacheHash =
+      PipelineDumper::generateHashForGlueShader({glueShaderIdentifier.size(), glueShaderIdentifier.data()});
+  return CacheAccessor(context, glueShaderCacheHash, compiler);
+}
+
+// =====================================================================================================================
+// Set the glue shader at glueIndex in the ELF linking with the data in the cache.  The data must be in the cache.
+//
+// @param elfLinker : The linker object for which the glue shaders are needed.
+// @param glueIndex : The index of the glue shader to be set.
+// @param cacheAccessor : The cache accessor that resulted from checking the caches for the glue shader.
+static void setGlueBinaryBlobFromCacheData(ElfLinker *elfLinker, uint32_t glueIndex,
```

Comment:
Better to use `unsigned `rather than `uint32_t`

Reply:
Done.